app/controllers/api/v0/highlights_controller.rb

```diff
@@ -48,6 +45,62 @@ def create
     render json: response_binding, status: :created
   end
 
+  swagger_path '/highlights' do
+    operation :get do
+      key :summary, 'Get selected highlights'
+      key :description, <<~DESC
+        Get selected highlight(s) for the owner.
+
+        Highlights can be filtered thru query parameters:  source_type,
+        source_parent_ids, and color.
+
+        Highlights are always paginated.  Pass in page and per_page to
+        request specific page, or omit the pagination params to get the defaults
+        of page = 1 and per_page of 15.
```

Comment:
Can this be `page = #{Api::V0::Bindings::GetHighlights::PAGING_DEFAULTS[:page]}` and similar for `per_page`?

Reply:
added to the TODO in this PR